arangod/Auth/UserManager.h

```diff
@@ -79,7 +79,7 @@ class UserManager {
 
   /// @brief reload user cache and token caches
   void triggerLocalReload() {
-    _globalVersion.fetch_add(1, std::memory_order_release);
+    _internalVersion.store(0, std::memory_order_release);
```

Comment:
`loadFromDB()` will repopulate the cache if `_internalVersion != _globalVersion`.
So this should work fine for any `_globalVersion != 0`.
And `_globalVersion` is initialized to `1` in all ctors. So unless we have an overflow from UINT64_MAX to 0 it should work.